src/develop/pixelpipe_hb.h

```diff
@@ -93,13 +89,10 @@ typedef struct dt_dev_pixelpipe_t
   float iscale;
   // dimensions of processed buffer
   int processed_width, processed_height;
-  // sensor saturation, propagated through the operations:
-  float processed_maximum[4];
 
-  // sensor pattern aka filters, propagated through the operations:
-  uint32_t filters; // Bayer demosaic pattern
-  /* filter for Fuji X-Trans images, only used if filters == 9u */
-  uint8_t xtrans[6][6];
+  // this one actually contains the expected output format,
+  // and should be modified by process*(), if necessary.
+  dt_iop_buffer_dsc_t dsc;
```

Comment:
why exactly do we need this here? sounds like asking for trouble having them on the per-iop pieces and on the full pipe, too? will this be overwritten as the pipe is processed? and then contain different data than the dsc_out on the piece? how does it work with tiling?